coursebook/modules/m1/Lesson_1.4_Collaboration_and_reproducibility.ipynb

```diff
@@ -0,0 +1,182 @@
+    "\n",
+    "\n",
+    "### Scrum\n",
+    "In software development, Scrum is a lightweight framework built following Agile principles. It requires a \"Scrum Master\" to foster an environment where:\n",
+    "\n",
+    "- A person (the \"product owner\") orders a specific product\n",
+    "\n",
+    "- A team focuses on a specific aspect of the work during a Sprint.\n",
+    "\n",
+    "- The team and its stakeholders inspect the results and adjust for the next Sprint.\n",
+    "\n",
+    "- Repeat\n",
+    "\n",
+    "The cadence of each iteration is very short, with Sprints of two to max four weeks, with clear start and finish dates. Sprints are punctuated by the sprint planning, sprint review, and retrospective meetings and peppered with daily scrum(standup) meetings. Such steps are lightweight and run on a continuous basis. \n",
+    "\n",
+    "When employing such concepts in a Research Data Science project, the focus will be on:\n",
+    "\n",
+    "1. Splitting the complex work that you want to achieve into small-scale specific tasks with a clear outcome. \n",
+    "2. Sprint planning with the researchers and domain experts (the equivalent of the \"product owner\") in short focused meetings (max 30 mins), to defining the goal of each sprint (which specific task you want to address) and the duration. Everyone needs to agree, so that expectations are aligned. \n",
+    "3. During the sprint it is important to have regular, short and effective checks with collaborators (as standups meetings) and regular touch-base with the researchers to highlight: what has been done, what we are planing to do, which blockers we have found.\n",
+    "4. At the end of each sprint it is essential to discuss results, difficulties and to improve the planning of the next sprint.\n",
+    "\n",
+    "### Kanban board\n",
+    "Kanban helps visualise your work, limit the amount of work-in-progress (WIP) and rapidly moving tasks from \"Doing\" to \"Done\". The core is a Kanban board, which is for instance available in GitHub under the tab \"Projects\" and you could use it to organise the issues and workflow related to a project and highlight the stage of a specific Sprint. Kanban boards have three main components: Visual signals (the issues), columns (the basic structure is \"To Do\", \"In Progress\" and \"Done\") and work-in-progress limits. Depending on the type of project, input and output of the board will be defined (who creates the issues for instance and who tests that the contribution is what was expected). The most important aspect of a Kanban board is that is shows clearly how your team works and the stage of the project.\n",
```

Comment:
It would be nice to show some examples of using a Kanban board in real REG projects during scoping (as the attendees will do this in the subsequent hands-on session).

Reply:
Still looking for a good example for this